arch/Kconfig

```diff
@@ -742,6 +747,16 @@ config FPU_SHARING
 
 endmenu
 
+menu "Floating Point Options"
```

Comment:
Why is this under "Floating Point Options"?
The DSP stuff we have is about Fixed Point.

Reply:
fixed